vision/api/Detect/Program.cs

```diff
@@ -318,6 +333,109 @@ private static object DetectDocText(Image image)
             return 0;
         }
 
+        // [START vision_async_detect_document_ocr]
+        private static object DetectDocument(string gcsSourceUri, string gcsDestinationUri)

```

Comment:
Consider updating method signature (and CLI) to something like this:

```cs
private static object DetectDocument(string gcsSourceUri, string destinationBucket, string destinationPrefix)
```

Mentioned below in [this comment](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/574/files#r202859816)